File: totaleconomy-core/src/main/java/com/ericgrandt/totaleconomy/data/entity/AccountEntity.java
package com.ericgrandt.totaleconomy.data.entity;

import java.math.BigDecimal;
import java.time.Instant;

public record AccountEntity(
    int id,
    String playerId,
    String currencyCode,
    BigDecimal balance,
    Instant createdAt
) {
}

File: totaleconomy-core/src/main/kotlin/com/ericgrandt/totaleconomy/data/entity/AccountEntity.kt
package com.ericgrandt.totaleconomy.data.entity

import java.util.UUID
import kotlin.time.Instant

data class AccountEntity(
    val id: Int,
    val playerId: UUID,
    val currencyCode: String,
    val balance: Double,
    val createdAt: Instant,
)
